src/Tag/stories.js

```diff
@@ -15,17 +13,18 @@ storiesOf('Tag')
       <Tag
         size={radios('Size', ['small', 'medium', 'big'], 'small')}
         variant={radios('Variant', ['invert', 'error', 'pro', 'success', 'warning'])}>
-        <span>{ text('Label', 'This is a tag') }</span>
+        <span>{text('Label', 'This is a tag')}</span>
       </Tag>
     </figure>
   ))
   .add('With a ButtonIcon', () => (
     <figure>
       <Tag
         size={radios('Size', ['small', 'medium', 'big'], 'small')}
-        variant={radios('Variant', ['invert', 'error', 'pro', 'success', 'warning'])}>
-        <span>{ text('Label', 'This is a tag') }</span>
-        <ButtonIcon><Icon size="small" name="close" /></ButtonIcon>
+        variant={radios('Variant', ['invert', 'error', 'pro', 'success', 'warning'])}
```

Comment:
We have a proposal to use the icon props in another component that use an icon. The goal of this is testing if is possible to use that prop and it's working fine.